pynecone/components/disclosure/transition.py

```diff
@@ -0,0 +1,70 @@
+"""A transition Component."""
+from typing import Union
+
+from pynecone.components.libs.chakra import ChakraComponent
+from pynecone.utils import imports
+from pynecone.vars import ImportVar, Var
+
+
+class Transition(ChakraComponent):
+    """Base componemt of all transitions."""
+
+    in_: Var[bool]
+
+    unmount_on_exit: Var[bool] = False  # type: ignore
+
+    def _get_imports(self) -> imports.ImportDict:
+        return imports.merge_imports(
+            super()._get_imports(),
+            {"@chakra-ui/react": {ImportVar(tag="useDisclosure")}},
+        )
+
+
+class Fade(Transition):
+    """Fade component cab be used show and hide content of your app."""
+
+    tag = "Fade"
+
+
+class ScaleFade(Transition):
+    """Fade component can be scaled and reverse your app."""
+
+    tag = "ScaleFade"
+
+    unmount_on_exit: Var[bool] = False  # type: ignore
```

Comment:
I think some props like this we don't need to explicitly set a default value. It will default to `None` and won't render as one of the React props (so will use the default React value).

Reply:
I understand. Thanks for your comments.